package.json

```diff
@@ -269,6 +269,12 @@
           "additionalProperties": false,
           "markdownDescription": "Whether to display Quick Fix actions to disable rules via `noqa` suppression comments."
         },
+        "ruff.showSyntaxErrors": {
+          "default": true,
```

Comment:
What was the default before. Did ruff always filter out syntax error diagnostics? I'm mainly asking to understand if this is a "breaking" change in the sense that the default changes

Reply:
No, Ruff would always show syntax error diagnostics.

Reply:
Good :) Unless someone used `ignore=E999` but that "breaking" change is already handled by Ruff's changelog

Reply:
Yeah, I assume you mean `ruff.ignore` server setting. Ruff would emit a warning in that case but it won't be displayed to the user, it'll be in the logs:
```
2024-06-27 12:34:00.546 [info] warning: Rule `E999` is deprecated and will be removed in a future release. Syntax errors will always be shown regardless of whether this rule is selected or not.
```
But, this won't work for `ignore` because it doesn't work in Ruff itself.